recipes-bsp/acpica/files/manipulate-fds-instead-of-FILE.patch

```diff
@@ -0,0 +1,71 @@
+From 540d80469e6a7dce6baf7214df90e86daffc5175 Mon Sep 17 00:00:00 2001
```

Comment:
No, let's drop these hacks. Update is okay, hacks are not okay for meta-acpi.

Reply:
I can try, but 'flex 2.6.3 no longer does that, leading to a compiler warning
and link error about yy_scan_string()."

Weird thing is sumo/thud/master have flex 2.6.0. So should be good. Unless they try to build against host (Ubuntu 18.10 has 2.6.4).

Reply:
I see, so, the patch should be upstreamed then.

Reply:
The patches are marked as:

> Upstream-Status: Inappropriate [embedded specific]\
> Signed-off-by: Patrick Ohly <patrick.ohly@intel.com>
>
However, I upgraded now to 20190215, tested building on Thud with the patches dropped. Also tested native on Ubuntu 18.10 (flex 2.6.4 ans bison 3.0.4) with no errors. 20190215 has acpi 6.3 and is in kernel 5.1-rc1.
This will appear in v3 of this PR.